pupil_src/shared_modules/pupil_detector_plugins/__init__.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import typing as T
+
+from .detector_base_plugin import PupilDetectorPlugin
+
+logger = logging.getLogger(__name__)
+
+
+def available_detector_plugins() -> T.Tuple[
+    PupilDetectorPlugin, T.List[PupilDetectorPlugin]
+]:
+    """Load and list available plugins, including default
+    
+    Returns:
+        T.Tuple[PupilDetectorPlugin, T.List[PupilDetectorPlugin]]
+        --  Return tuple of default plugin, and list available plugins.
+            Default is required to be in the list of available plugins.
+    """
+    from .detector_dummy_plugin import DetectorDummyPlugin
+    from .detector_2d_plugin import Detector2DPlugin
+    from .detector_3d_plugin import Detector3DPlugin
```

Comment:
Can we import this at the top?
Or would this cause problems with import order?
Would also be good to expose those detectors via the module I think.